web/src/util.js

```diff
@@ -188,3 +188,9 @@ export function formatBiosampleDepth(depthAnnotation, depth) {
   }
   return formattedStr;
 }
+
+/**
+ * Base URL (without a trailing slash) at which the user can access
+ * the same ORCID environment being used by the backend API.
+ */
+export const ORCID_BASE_URL = process.env.NMDC_ORCID_BASE_URL || "https://orcid.org";
```

Comment:
```suggestion
export const ORCID_BASE_URL = process.env.NMDC_ORCID_BASE_URL || 'https://orcid.org';
```

Reply:
GitHub won't allow me to commit this suggestion (maybe because there's no branch dedicated to this PR). So, I'll fix this in `main`.